cfy_cluster_setup/cfy_cluster_config_files/cfy_cluster_config.yaml

```diff
@@ -84,7 +83,7 @@ existing_vms:
       key_path: ''  # Need to be supplied if ca_cert_path was supplied
 
 
-# If the credentials are not specified, random ones will be used and written to a dedicated file
+# If the credentials are not specified, random ones will be used and written to ./secret_credentials.yaml
```

Comment:
Is there a way to make this path a jinja template key?

Reply:
Jinja doesn't think that's a comment, so yes.